### PR TITLE
Manage Access Tokens Cypress test

### PR DESCRIPTION
**Pull request by kkorchak**
Test steps:
1. Create access token
2. Verify token created successfully
3. Revoke access token
4. Verify token removed successfully

## Checklist

- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: datahub-web-react/src/app/settings/AccessTokenModal.tsx [hunk 1]
```diff
@@ -60,7 +60,7 @@ export const AccessTokenModal = ({ visible, onClose, accessToken, expiresInText
             onCancel={onClose}
             footer={
                 <>
-                    <Button id="createTokenButton" onClick={onClose}>
+                    <Button id="createTokenButton" onClick={onClose} data-testid="close-button">
```
Comment (jjoyce0510) [inline datahub-web-react/src/app/settings/AccessTokenModal.tsx:63]: can we name this access-token-modal-close-button?

usually we like to have data-testid to be more specific

File: datahub-web-react/src/app/settings/AccessTokenModal.tsx [hunk 2]
```diff
@@ -81,7 +81,7 @@ export const AccessTokenModal = ({ visible, onClose, accessToken, expiresInText
                 <ModalSectionHeader strong>Token</ModalSectionHeader>
                 <ModalSectionParagraph>{expiresInText}</ModalSectionParagraph>
                 <Typography.Paragraph copyable={{ text: accessToken }}>
-                    <pre>{accessToken}</pre>
+                    <pre data-testid="access-token">{accessToken}</pre>
```
Comment (jjoyce0510) [inline datahub-web-react/src/app/settings/AccessTokenModal.tsx:84]: let's call this access-token-value

File: datahub-web-react/src/app/settings/CreateTokenModal.tsx [hunk 3]
```diff
@@ -117,10 +117,15 @@ export default function CreateTokenModal({ currentUserUrn, visible, onClose, onC
...
-                        <Button onClick={onModalClose} type="text">
+                        <Button onClick={onModalClose} type="text" data-testid="cancel-button">
                             Cancel
                         </Button>
-                        <Button id="createTokenButton" onClick={onCreateNewToken} disabled={createButtonEnabled}>
+                        <Button
+                            id="createTokenButton"
+                            onClick={onCreateNewToken}
+                            disabled={createButtonEnabled}
+                            data-testid="create-token-button"
```
Comment (jjoyce0510) [inline datahub-web-react/src/app/settings/CreateTokenModal.tsx:127]: create-access-token-button

File: datahub-web-react/src/app/settings/AccessTokenModal.tsx (hunk 1 above)
Comment (kkorchak) [reply datahub-web-react/src/app/settings/AccessTokenModal.tsx:63]: sure

File: datahub-web-react/src/app/settings/CreateTokenModal.tsx [hunk 4]
```diff
@@ -117,10 +117,15 @@ export default function CreateTokenModal({ currentUserUrn, visible, onClose, onC
                 onCancel={onModalClose}
                 footer={
                     <>
-                        <Button onClick={onModalClose} type="text">
+                        <Button onClick={onModalClose} type="text" data-testid="cancel-button">
```
Comment (jjoyce0510) [inline datahub-web-react/src/app/settings/CreateTokenModal.tsx:120]: cancel-create-access-token-button

File: datahub-web-react/src/app/settings/CreateTokenModal.tsx [hunk 5]
```diff
@@ -148,18 +153,18 @@ export default function CreateTokenModal({ currentUserUrn, visible, onClose, onC
                             ]}
                             hasFeedback
                         >
-                            <Input placeholder="A name for your token" />
+                            <Input placeholder="A name for your token" data-testid="token-name" />
```
Comment (jjoyce0510) [inline datahub-web-react/src/app/settings/CreateTokenModal.tsx:156]: create-access-token-name

File: datahub-web-react/src/app/settings/CreateTokenModal.tsx [hunk 6]
```diff
@@ -148,18 +153,18 @@ export default function CreateTokenModal({ currentUserUrn, visible, onClose, onC
...
                         </Form.Item>
                     </Form.Item>
                     <Form.Item label={<Typography.Text strong>Description</Typography.Text>}>
                         <Typography.Paragraph>An optional description for your new token.</Typography.Paragraph>
                         <Form.Item name="description" rules={[{ whitespace: true }, { min: 1, max: 500 }]} hasFeedback>
-                            <Input placeholder="A description for your token" />
+                            <Input placeholder="A description for your token" data-testid="token-description" />
```
Comment (jjoyce0510) [inline datahub-web-react/src/app/settings/CreateTokenModal.tsx:162]: create-access-token-description

File: datahub-web-react/src/app/settings/CreateTokenModal.tsx [hunk 7]
```diff
@@ -148,18 +153,18 @@ export default function CreateTokenModal({ currentUserUrn, visible, onClose, onC
...
                         </Form.Item>
                     </Form.Item>
                     <ExpirationSelectContainer>
                         <Typography.Text strong>Expires in</Typography.Text>
-                        <Form.Item name="duration" noStyle>
+                        <Form.Item name="duration" data-testid="token-duration" noStyle>
```
Comment (jjoyce0510) [inline datahub-web-react/src/app/settings/CreateTokenModal.tsx:167]: create-access-token-duration

File: datahub-web-react/src/app/settings/CreateTokenModal.tsx (hunk 3 above)
Comment (kkorchak) [reply datahub-web-react/src/app/settings/CreateTokenModal.tsx]: done

File: datahub-web-react/src/app/settings/CreateTokenModal.tsx (hunk 4 above)
Comment (kkorchak) [reply datahub-web-react/src/app/settings/CreateTokenModal.tsx]: done

File: datahub-web-react/src/app/settings/CreateTokenModal.tsx (hunk 6 above)
Comment (kkorchak) [reply datahub-web-react/src/app/settings/CreateTokenModal.tsx]: done

File: datahub-web-react/src/app/settings/CreateTokenModal.tsx (hunk 7 above)
Comment (kkorchak) [reply datahub-web-react/src/app/settings/CreateTokenModal.tsx]: done

**Review by jjoyce0510 — APPROVED** [review]

**Pull request merged**